tests/models/nn/transformers/test_bert4rec.py

```diff
@@ -114,6 +123,38 @@ def get_trainer() -> Trainer:
 
         return get_trainer
 
+    @pytest.fixture
+    def get_custom_trainer_func(self) -> TrainerCallable:
+        def get_trainer_func(**kwargs: Unpack[KwargsSpec]) -> Trainer:
```

Comment:
Since this is a test, we should write it the way user would use it.
And in this case it should be `def get_trainer_func(max_epochs: int,  accelerator: str) -> Trainer:`
Because here he wants hit triner func to change variants of max_epochs and accelerator and accept them from model config.